memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
+
+## Quick Start
+
+This section describes how to run YCSB on memcached.
+
+### 1. Install and start memcached service on the host(s)
+
+### 2. Install Java and Maven
+
+### 3. Set up YCSB
+
+Git clone YCSB and compile:
+
+    git clone http://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+    mvn -pl com.yahoo.ycsb:memcached-binding -am clean package
+
+### 4. Provide memcached Connection Parameters
+
+Set the list of hosts in the workload you plan to run.
+
+- `memcached.hosts`
+
+This is a comma-separated list of hosts providing the memcached interface. You
+can use IPs or hostnames. The port is optional and defaults to the memcached
+standard port of `11211` if not specified.
+
+Optional parameters are listed in
+[`conf/memcached.properties`](conf/memcached.properties).
```

Comment:
Having an example conf file is great, in addition can you please document the memcached specific properties at the end of this README? The mongodb module also has a good example of how to do that.